CUSTESC-48297 - clarify available filters depends on node (#19402)
cloudflare/cloudflare-docs: src/content/docs/analytics/graphql-api/features/filtering.mdx

* accounts
* tables (datasets)

> Which filters are available depends on the queried node. For example, some nodes may only support a zoneTag filter, some only an accountTag filter, and some may allow both.

Reviewer: ```suggestion
The availability of filters depends on the queried node. For example, some nodes may support only a zoneTag filter, others only an accountTag filter, and some may support both.
```
